### DISCUSSION TITLE
Live search drop down menu with google sheets

### DISCUSSION THREAD

**Discussion by gigido** [twbs/bootstrap#36224] [Q&A]
Hi I have an html dropdown menu that I am populating based on the values in my google sheet. When I add the "data-live-search="true" to the form nothing loads. Any ideas on what I would need to do to make both things work together?
Thanks.

**Comment by gerardnico** [comment]
You are at the wrong place it seems.

You search attribute is from

https://github.com/snapappointments/bootstrap-select
